File: src/main/java/com/jessica/guess/GameFor.java
package com.jessica.guess;


import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
//        int sum =0;
//        for(int i=5;i>-1;i--){
//            System.out.println(i);
//        }
        int secret =new Random().nextInt(10)+1;
        Scanner scanner=new Scanner(System.in);
        for(int i=1;i<=4;i++){
            int number=scanner.nextInt();
            System.out.println("time:"+ number);
            if(number>secret){
            System.out.println("lower");
        }else if(number<secret){
            System.out.println("higher");
        }else{
            System.out.println("guess the number"+number);
            break;
        }
    }
    }
}

File: src/main/kotlin/com/student/GameFor.kt
package com.student

import java.util.*

class GameFor {
}

fun main() {
    val secret = Random().nextInt(10)+1
    println(secret)
    val scanner=Scanner(System.`in`)
    for(i in 1..4){
        print("please enetr a number(${i}/4):")
        var number = scanner.nextInt()
        print("第${i}次$number")
        if(number>secret){
            println("lower")
        }else if (number<secret){
            println("higher")
        }else {
            println("Great,the number is" + secret)
            break;
        }
    }    }